apis/medusa/v1alpha1/cassandrabackup_types.go

```diff
@@ -45,6 +45,12 @@ type CassandraBackupSpec struct {
 	// +kubebuilder:validation:Enum=differential;full;
 	// +kubebuilder:default:=differential
 	Type BackupType `json:"backupType,omitempty"`
+
+	// Specifies if the object was created by a synchronization job.
+	// If so, reconciliation will be skipped.
+	// +kubebuilder:default:=false
+	// +kubebuilder:validation:optional
+	CreatedBySync bool `json:"createdBySync,omitempty"`
```

Comment:
I don't like this idea. I think all objects should be reconciled and synced to maintain state as Kubernetes is our user interface.

Reply:
I'll shortly send a new PR which separates the action of taking a backup from the backup itself, introducing the MedusaBackupJob and MedusaBackup APIs.